packages/bridging/src/providers/bungee/createBungeeDepositCall.test.ts

```
@@ -0,0 +1,193 @@
+import { setGlobalAdapter } from '@cowprotocol/sdk-common'
+import { SupportedChainId } from '@cowprotocol/sdk-config'
+import { OrderKind } from '@cowprotocol/sdk-order-book'
+import { createAdapters } from '../../../tests/setup'
+import { QuoteBridgeRequest } from '../../types'
+import { BUNGEE_APPROVE_AND_BRIDGE_V1_ABI } from './abi'
+import { createBungeeDepositCall } from './createBungeeDepositCall'
+import { BungeeBridge, BungeeQuoteWithBuildTx } from './types'
+import { decodeAmountsBungeeTxData, toBridgeQuoteResult } from './util'
+
+export const CCTP_V2_TX_DATA =
+  '0x000001a63ca7f5bc0000000000000000000000000000000000000000000000000000000005f5e1000000000000000000000000000000000000000000000000000000000000002713000000000000000000000000664b591ab924c6bb2caca533ed702386934a11d6000000000000000000000000833589fcd6edb6e08f4c7c32d4f71b54bda029130000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001e8480000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000003e8'
```

Biome (2.4.6)

[error] Do not export from a test file.

(lint/suspicious/noExportsInTest)